gallery/plot_visualization_utils.py

```diff
@@ -43,8 +43,9 @@ def show(imgs):
 
 dog1_int = read_image(str(Path('assets') / 'dog1.jpg'))
 dog2_int = read_image(str(Path('assets') / 'dog2.jpg'))
+dog_list = [dog1_int, dog2_int]
 
-grid = make_grid([dog1_int, dog2_int, dog1_int, dog2_int])
+grid = make_grid(dog_list)
```

Comment:
Not sure why we were showing twice the picture of each dog previously.

Reply:
I'd be in favour of removing this section about `make_grid` entirely. We shouldn't encourage our users to use it anymore, there are much better alternatives

Reply:
Sounds good to me, let's do this on a separate PR to explore what the better recommended alternative is. cc @oke-aditya